packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,514 @@
+        </Menu>
+      </MenuTrigger>
+      <span style={{'margin-left': '8px'}}>Current open state: {state.toString()}</span>
+    </div>
+  );
+}
+```
+
+### onAction
+
+Menu accepts an `onAction` handler which is triggered whenever a Menu Item is selected.
+
+The example below uses the `onAction` to update text above with Menu with the last selected Item.
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(false);
+
+  return (
+    <div>
+      <div>Edit (onAction: {state.toString()})</div>
+      <Menu
+        onAction={(value) => setState(value)}
+        width="200px">
+        <Item uniqueKey="cut">Cut</Item>
+        <Item uniqueKey="copy">Copy</Item>
+        <Item uniqueKey="paste">Paste</Item>
+      </Menu>
+    </div>
+  );
+}
+```
+
+## Props
+
+### MenuTrigger
+
+<PropTable component={docs.exports.MenuTrigger} links={docs.links} />
+
+### Menu
+
+<PropTable component={docs.exports.Menu} links={docs.links} />
+
+## Behavioral Options
+
+### Align (MenuTrigger)
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Placement)
+
+```tsx example
+<MenuTrigger align="start">
+  <ActionButton>
+    placement align=start
+  </ActionButton>
+  <Menu>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end">
+  <ActionButton>
+    placement align=end
+  </ActionButton>
+  <Menu>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+### Direction (MenuTrigger)
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Placement)
+
+```tsx example
+<MenuTrigger direction="bottom" shouldFlip={false}>
+  <ActionButton>
+    placement direction=bottom
+  </ActionButton>
+  <Menu>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+<MenuTrigger direction="top" shouldFlip={false}>
+  <ActionButton>
+    placement direction=top
+  </ActionButton>
+  <Menu>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+### Autofocus (Menu)
+
+Applying `autoFocus` to the Menu of the MenuTrigger sets focus to a Menu Item
+within the Menu upon opening.
+
+These examples demonstrate how to use `autoFocus` to set whether or not
+the selected Menu Item should be automatically focused when the Menu is opened.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+    autofocus
+  </ActionButton>
+  <Menu
+    selectionMode="single"
+    selectedKeys={['copy']}
+    autoFocus>
+    <Item uniqueKey="cut">Cut</Item>
+    <Item uniqueKey="copy">Copy</Item>
+    <Item uniqueKey="paste">Paste</Item>
+  </Menu>
+</MenuTrigger>
+<MenuTrigger>
+  <ActionButton>
+    autofocus=false
```

Comment:
given that most examples in this page don't have autofocus, do we want to omit this one?
also, autoFocus camel case